Find includes and libs in the  tree, install there.
Use cp instead of 'install -t', which does not work on Mac OS X.
Disable plrs build.

--- patches/makefile.orig	2015-06-14 18:06:18.000000000 -0700
+++ patches/makefile	2015-06-14 18:06:18.000000000 -0700
@@ -25,8 +25,8 @@
 #Select a path below to give location of gmp library
 
 #cygwin
-INCLUDEDIR = /usr/include
-LIBDIR     = /usr/lib
+INCLUDEDIR = ${SAGE_LOCAL}/include
+LIBDIR     = ${SAGE_LOCAL}/lib
 
 #linux at mcgill with gmp version 3
 #INCLUDEDIR = /usr/local/include
@@ -45,7 +45,7 @@
 LIBRARIES=$(LIB) $(SHLIB) $(SHLINK)
 
 # where to install binaries, libraries, include files
-prefix := /usr/local
+prefix := ${SAGE_LOCAL}
 
 # Shared library
 SONAME ?=liblrsgmp.so.0
@@ -107,19 +107,19 @@
 
 install-shared: all-shared install-common
 	 mkdir -p $(DESTDIR)${prefix}/lib
-	 install -t $(DESTDIR)${prefix}/lib $(SHLIB)
+	 cp $(SHLIB)  $(DESTDIR)${prefix}/lib 
 	 cd $(DESTDIR)${prefix}/lib && ln -sf $(SHLIB) $(SHLINK)
 	 cd $(DESTDIR)${prefix}/lib && ln -sf $(SHLIB) $(SONAME)
 
 install-static: all-static install-common
 	 mkdir -p $(DESTDIR)${prefix}/lib
-	 install -t $(DESTDIR)${prefix}/lib $(LIB)
+	 cp $(LIB) $(DESTDIR)${prefix}/lib 
 
 install-common:
 	 mkdir -p $(DESTDIR)${prefix}/bin
-	 install -t $(DESTDIR)${prefix}/bin $(BINARIES)
+	 cp $(BINARIES) $(DESTDIR)${prefix}/bin 
 	 mkdir -p $(DESTDIR)${prefix}/include
-	 install -t $(DESTDIR)${prefix}/include lrslib.h lrsgmp.h
+	 cp lrslib.h lrsgmp.h $(DESTDIR)${prefix}/include 
 
 fourier:	fourier.c lrslib.h lrslib.c lrsgmp.h lrsgmp.c
 	gcc -O3 -DTIMES -DSIGNALS  -DGMP -I${INCLUDEDIR} fourier.c lrslib.c lrsgmp.c -L${LIBDIR}  -lgmp -o fourier
@@ -142,6 +142,17 @@
 		gcc -O3 -o setnash2 setupnash2.c lrslib.c lrsmp.c
 		gcc -O3 -o 2nash 2nash.c
 
+check:
+	lrs ext/test/cut16_11.ext > testout1
+	echo `grep -G Volume testout1` > testout1
+	echo "*Volume= 32768/14175" > testout2
+	diff testout1 testout2
+	lrs ine/test/cyclic17_8.ine > testout1
+	echo `grep -G Totals testout1` > testout1
+	echo "*Totals: vertices=935 rays=0 bases=935 integer_vertices=0" > testout2
+	diff testout1 testout2
+	rm testout1
+	rm testout2
 
 clean:		
-		rm -f $(BINARIES) $(LIBRARIES) plrs *.o *.exe
+		rm -f $(BINARIES) $(LIBRARIES) plrs *.o *.exe testout1 testout2
